content/docs/actors/development/actor_config.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Actor config
+description: Learn how to write the main actor config in the `.actor/actor.json` file.
+paths:
+    - actor/development/actor-config
+    - actors/development/actor-config
```

Comment:
You don't need the first line - those are just for redirects for old files that used to be in the `/actor` folder
```suggestion
    - actors/development/actor-config
```